analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/ComplianceAnalyzerPlugin.java

```diff
@@ -0,0 +1,278 @@
+                // Deploy ConfigMap
+                V1ConfigMap configMap = Yaml.loadAs(new File("docker/master-config.yaml"), V1ConfigMap.class);
+                V1ConfigMap deployedConfigMap = new CoreV1Api().createNamespacedConfigMap(K8S_NAMESPACE, configMap, null, null, null);
+                System.out.println("Deployed ConfigMap: " + deployedConfigMap);
+
+                // Deploy Service
+                V1Service service = Yaml.loadAs(new File("docker/service.yaml"), V1Service.class);
+                V1Service deployedService = new CoreV1Api().createNamespacedService(K8S_NAMESPACE, service, null, null, null);
+                System.out.println("Deployed Service: " + deployedService);
+
+                // Define and apply a qmstr kubernetes job
+                Yaml.addModelMap("v1", "Job", V1Job.class);
+
+                File file = new File(PATCHED_JOB_FILE_NAME);
+                V1Job yamlJob = (V1Job) Yaml.load(file);
+                logger.info("Job definition: " + yamlJob);
+                BatchV1Api api = new BatchV1Api();
+
+                V1Job result = api.createNamespacedJob(K8S_NAMESPACE, yamlJob, null, null, null);
+                System.out.println(result);
+
+                // delete job
+                /*
+                V1Status deleteResult = api.deleteNamespacedJob(yamlJob.getMetadata().getName(),
+                KUBERNETES_NAMESPACE, null, null, null, null, null, new V1DeleteOptions());
+                logger.info(deleteResult);
+                s*/
+            } catch (IOException ex) {
+                logger.info(ex.toString());
+                logger.info("Could not find file: " + PATCHED_JOB_FILE_NAME);
+            } catch (ApiException ex) {
+                logger.error("Status code: " + ex.getCode());
+                logger.error("Reason: " + ex.getResponseBody());
+                logger.error("Response headers: " + ex.getResponseHeaders());
+                ex.printStackTrace();
+            }
+        }
+
+        @Override
+        public Optional<String> produce() {
+            // dump output for now
+            return Optional.of("License Compliance Plugin finished successfully");
+        }
+
+        @Override
+        public String getOutputPath() {
+            return this.outputPath;
+        }
+
+         @Override
+         public String name() {
+             return "License Compliance Plugin";
+         }
+
+          @Override
+         public String description() {
+             return "License Compliance Plugin."
+             + "Consumes Repository Urls from Kafka topic,"
+             + " connects to cluster and starts a Kubernetes Job."
+             + " The Job spins a qmstr process which detects the project's"
+             + " license compliance and compatibility."
+             + " Once the Job is done the output is written to another Kafka topic.";
+ }
+          @Override
+         public String version() {
+             return "0.0.1";
+         }
```

Comment:
Indentation is off

Reply:
Solved in https://github.com/fasten-project/fasten/commit/7975246450da52793713310dc6123c14c876daf5.